landing/index.html

```diff
@@ -65,20 +65,22 @@ <h2 class="h2 margin-list__item">
                         </h2>
                 
                         <div class="margin-list__item paragraph paragraph_large">
-                            Нашим продуктом є Open Data Platform, де представлені глобальні статистичні дані
+                            Open Data Platform — не про сухі дані.
+                            Це візуалізований конструктор даних для доступу до статистичної інформації, необхідної бізнесу для прийняття рішень.
+                            Також, платформа є сервісом зручного доступу до даних за допомогою Rest API для розробників програмного забезпечення.
                         </div>
 
                         <div class="margin-list__item form-subscribe">
                             <input class="input input_large form-subscribe__input" placeholder="user@email.com">
                             <button class="btn-primary btn-primary_large form-subscribe__action">Підписатися</button>
                         </div>
-                  </div>
+                </div>
 
                 <div class="col-lg-6 landing-image">
-                    <img  class="landing-image__image" src="static/img/dashboard.png">
+
+                    <a href="https://dp.dataocean.us/system/home/" target="_blank"><button><img  class="landing-image__image" src="static/img/dashboard1.png"></button></a>
```

Comment:
unessesary button inside anchor.
and make this line shorter, you can add some line breaks.